woocommerce/woocommerce-ios
# Add Events for "Jetpack Required" Screen

When I was analyzing the login data (p91TBi-5Yh-p2), I found out that some events for the "Jetpack required" screen are not tracked by iOS. These are tracked by Android. 

## Changes

I added tracking for the events to match Android's tracked events. 

![Image 2021-09-15 16-51-49](https://user-images.githubusercontent.com/198826/133522135-3f8026a5-ad83-4ca6-a775-88121672b595.jpg)

![Image 2021-09-15 16-56-42](https://user-images.githubusercontent.com/198826/133523213-317884a3-bc41-4729-a23b-1bf884f42e05.jpg)


## Testing

1. Log out.  😛 
2. In the login screen, tap Enter Your Store Address.
3. Enter a site URL that has no Jetpack. 
4. Confirm `login_jetpack_required_screen_viewed` is tracked. 
5. Tap See Instructions.
6. Confirm `login_jetpack_required_view_instructions_button_tapped` is tracked. 
7. Dismiss the window. Unless you want to read the page. Feel free! 🤪 
8. Tap What Is Jetpack?
9. Confirm `login_what_is_jetpack_help_screen_viewed` is tracked. 
10. Tap Learn more about Jetpack.
11. Confirm `login_what_is_jetpack_help_screen_learn_more_button_tapped` is tracked. 
12. Dismiss the browser window.
13. Tap Continue.
14. Confirm `login_what_is_jetpack_help_screen_ok_button_tapped` is tracked. 

If you can, please compare with Android's events too. 

## Reviewing

🏳️ Please nitpick me to the death. I am rusty. 😁 I'm not in a hurry to get this merged. 

Only 1 reviewer is needed but anyone can review.

## Author Checklist

- [x] Update Tracks events spreadsheet
